bigtable/snippets/garbagecollection/garbagecollection_test.go

```diff
@@ -16,37 +16,33 @@ package garbagecollection
 
 import (
 	"bytes"
+	"context"
+	"log"
+	"os"
 	"strings"
 	"testing"
 
 	"cloud.google.com/go/bigtable"
-	"context"
-	"log"
-	"os"
 )
 
 func TestGarbageCollection(t *testing.T) {
 	ctx := context.Background()
 	project := os.Getenv("GOLANG_SAMPLES_BIGTABLE_PROJECT")
 	instance := os.Getenv("GOLANG_SAMPLES_BIGTABLE_INSTANCE")
 	if project == "" || instance == "" {
-		t.Skip("Skipping bigtable integration test. Set GOLANG_SAMPLES_BIGTABLE_PROJECT " +
-			"and GOLANG_SAMPLES_BIGTABLE_INSTANCE.")
+		t.Skip("Skipping bigtable integration test. Set GOLANG_SAMPLES_BIGTABLE_PROJECT and GOLANG_SAMPLES_BIGTABLE_INSTANCE.")
 	}
 	adminClient, err := bigtable.NewAdminClient(ctx, project, instance)
 
 	tableName := "gc-table"
-	if err = adminClient.DeleteTable(ctx, tableName); err != nil {
-		log.Printf("Could not delete table %s: %v", tableName, err)
-	}
+	adminClient.DeleteTable(ctx, tableName)
 
 	if err := adminClient.CreateTable(ctx, tableName); err != nil {
 		log.Fatalf("Could not create table %s: %v", tableName, err)
```

Comment:
Should use `t.Fatal`. `log.Fatal` stops the entire test suite rather than just this test.

Reply:
Done.